### PR TITLE
Ilverify fixes

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

**Review by jashook — APPROVED** [review]
lgtm thank you for working on this.

File: src/coreclr/src/tools/ILVerification/ILVerification.csproj [hunk 1]
```diff
@@ -4,300 +4,12 @@
...
-      <Link>TypeSystem\Common\LocalVariableDefinition.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\System\FormattingHelpers.cs">
-      <Link>..\Common\System\FormattingHelpers.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\Common\TypeSystemConstraintsHelpers.cs">
-      <Link>TypeSystem\Common\TypeSystemConstraintsHelpers.cs</Link>
-    </Compile>
-  </ItemGroup>
+  <Import Project="ILVerification.projitems" />
```
Comment (safern) [inline src/coreclr/src/tools/ILVerification/ILVerification.csproj:12]: In order to open this correctly in VS, you'll need to add a .shproj just how we do with System.Private.CoreLib:

https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.shproj

**Comment by jashook** [comment]
outerloop started https://dev.azure.com/dnceng/public/_build/results?buildId=671115&view=results

File: src/coreclr/src/tools/ILVerification/ILVerification.csproj (hunk 1 above)
Comment (jkotas) [reply src/coreclr/src/tools/ILVerification/ILVerification.csproj:12]: What do I get by doing that? As far as I can tell, the current VS is perfectly happy without it.

Comment (safern) [reply src/coreclr/src/tools/ILVerification/ILVerification.csproj:12]: Does it show all the cs files? Maybe it is just needed for different subtrees. I don't recall why we needed to do it for S.P.CoreLib. @ViktorHofer do you remember?

Comment (jkotas) [reply src/coreclr/src/tools/ILVerification/ILVerification.csproj:12]: > Does it show all the cs files? 

Yes: image

Comment (safern) [reply src/coreclr/src/tools/ILVerification/ILVerification.csproj:12]: Ok then disregard my comment. I’ll let @ViktorHofer explain why it was needed for S.P.CoreLib.

Comment (ViktorHofer) [reply src/coreclr/src/tools/ILVerification/ILVerification.csproj:12]: the shproj for CoreLib was necessary as the projitems file is located in a different directory tree. VS refused to load without it.

Comment (safern) [reply src/coreclr/src/tools/ILVerification/ILVerification.csproj:12]: Awesome, so then I did remember sort of correctly after @jkotas told me it worked fine. Thanks @ViktorHofer for solving the mystery 😄

**Pull request merged**